### PR TITLE
Add client method for coordinate recon, which resolves latitude & longitude to place DCIDs. 

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

**Comment by Spaceenter** [comment]
NOTE: In the integration test, the 1st test case doesn't have geoId/06085 (Santa Clara county) in the golden file, it's a bug in the cache data (not a bug in this PR), and it's already fixed in cl/409441009 - after next cache build, I'll update the golden file.  @pradh @shifucun

**Review by pradh — COMMENTED** [review]
First pass... will look at golang file in a bit.

could you add description and a more meaningful title for the PR?

File: proto/recon.proto [hunk 1]
```diff
@@ -45,6 +45,13 @@ message EntityPair {
   EntitySubGraph entity_two = 2;
 }
 
+message PlaceCoordinate {
+  double latitude = 1;
+  double longitude = 2;
```
[the remark this answers is not in the document]
Comment (Spaceenter) [reply proto/recon.proto]: Done.

File: proto/recon.proto [hunk 2]
```diff
@@ -45,6 +45,13 @@ message EntityPair {
...
+  // This field can be empty if it's part of the request message.
+  repeated string place_ids = 3;
```
[the remark this answers is not in the document]
Comment (Spaceenter) [reply proto/recon.proto]: Done.

File: test/integration/golden_response/resolve_coordinates/result.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      "latitude": 32.41,
+      "longitude": -102.11,
+      "placeIds": [
+        "geoId/sch4825790",
+        "geoId/48317950100",
+        "geoId/4831793835",
+        "geoId/48317",
+        "geoId/4811",
+        "geoId/48",
+        "country/USA"
```
[the remark this answers is not in the document]
Comment (Spaceenter) [reply test/integration/golden_response/resolve_coordinates/result.json:23]: I see.
Two ways to do it:
1. Add it to the coordinate recon cache result.
2. Query PropVal cache to get the place type.

If #1, then obviously needs cache change. 
WDYT?

In any case, added a TODO to coordinate.go.

**Review by n-h-diaz — APPROVED** [review]
approving to unblock Wei since Bo and Prashanth are OOO this week and this code is not being used 
please address their comments when they return
thanks!

**Comment by Spaceenter** [comment]
Submitting for now to allow for further development on top of this. Since the code is not used, so there is no risk. Will have Bo or Shanth to look after their vacation, and can make more changes then. Thanks.

**Pull request merged**